internal/client/grpc/connect.go

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package grpc
+
+import (
+	"io"
+	"time"
+
+	jaegerClient "github.com/mainflux/mainflux/internal/client/jaeger"
+	"github.com/mainflux/mainflux/pkg/errors"
+	"github.com/opentracing/opentracing-go"
+	gogrpc "google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+var (
+	errGrpcConnect = errors.New("failed to connect to grpc server")
+	errJaeger      = errors.New("failed to initialize jaeger ")
+	errGrpcClose = errors.New("failed to close grpc connection")
+	errJaegerClose = errors.New("failed to close jaeger connection")	
+)
+
+type Config struct {
+	ClientTLS bool          `env:"CLIENT_TLS"    envDefault:"false"`
+	CACerts   string        `env:"CA_CERTS"      envDefault:""`
+	URL       string        `env:"URL"           envDefault:""`
+	Timeout   time.Duration `env:"TIMEOUT"       envDefault:"1s"`
+}
+
+type ClientHandler interface {
+	Close() error 
+	IsSecure() bool
+	Secure()string
```

Comment:
I know that this is used to simplify the logging, but it's not necessary. We can simply use `IsSecure` and log it as `true` or `false`, no need for nice print like `with(out) TLS`.